tools/cbmc/proofs/HTTP/IotHttpsClient_ReadHeader/Makefile.json

```diff
@@ -6,6 +6,10 @@
   "MAX_ACCEPTED_SIZE": 20,
   "STRNCPY_UNWIND": "__eval {MAX_ACCEPTED_SIZE} + 1",
 
+  # This is the length of the user data after the header in a buffer.
+  # This can be set to any size.
+  "USER_DATA_SIZE": "1000",
```

Comment:
Same comments as above: suggest 3 different configurations, suggest not defining a variable here